Readme.md

```diff
@@ -6,7 +6,7 @@ If you find our tools useful, then please consider supporting ENIGMA on Patreon!
 
 Official Website: http://enigma-dev.org
 
-ENIGMA, the Extensible Non-Interpreted Game Maker Augmentation, is an open-source, cross-platform game development environment based on the popular software, Game Maker. Its intent is to provide users with a quality game creation tool and a bridge between high- and low-level programming languages. It can be used either through an IDE, such as its sister project, [LateralGM](https://github.com/IsmAvatar/LateralGM), or its up-and-coming custom frontend, [RadialGM](https://github.com/enigma-dev/RadialGM), or also through its command line interface, [emake](CommandLine/emake).
```

Comment:
This hyphen's correct, actually; it's a style thing. Another example is [this Wikipedia article](https://en.wikipedia.org/wiki/High-_and_low-level).

Reply:
sure i will make this correct in next pull request, actually i am a newbie 😜